src/GraphQL/Schemas/Primary/Substrate/Mutations/BatchTransferMutation.php

```diff
@@ -112,61 +113,52 @@ function ($recipient) use ($blockchainService, $walletService) {
             }
         );
 
+        // Hard-coded to false until we add support for this feature back into the mutations.
+        $continueOnFailure = false;
+        $encodedData = $serializationService->encode($continueOnFailure ? 'Batch' : $this->getMutationName(), static::getEncodableParams(
+            collectionId: $args['collectionId'],
+            recipients: $recipients->toArray(),
+            continueOnFailure: $continueOnFailure
+        ));
+
         return Transaction::lazyLoadSelectFields(
-            $transactionService->store(
-                [
-                    'method' => $this->getMutationName(),
-                    'encoded_data' => $this->resolveBatch($args['collectionId'], $recipients, false, $serializationService),
-                    'idempotency_key' => $args['idempotencyKey'] ?? Str::uuid()->toString(),
-                    'deposit' => $this->getDeposit($args),
-                    'simulate' => $args['simulate'],
-                ],
-                signingWallet: $this->getSigningAccount($args)
-            ),
+            $this->storeTransaction($args, $encodedData),
```

Comment:
Does this affect the beam part as well https://github.com/enjin/platform-beam/blob/master/src/Commands/BatchProcess.php#L211?

Reply:
Also what about platform-fuel-tanks library, does it need update?

Reply:
> Does this affect the beam part as well https://github.com/enjin/platform-beam/blob/master/src/Commands/BatchProcess.php#L211?

I think we can leave the beam implementation as-is for now, the only meaningful change would be to use the new TransactionSerializer facade where it encodes that data.  The`StoresTransactions` trait is specifically for the transaction mutations at the moment as it uses the `getMutationName()` function available on the mutation, but not in the Beam BatchProcess command.

Reply:
lgtm but I'm a bit hesitant to have this on the next release, not enough time for testing.